mreg/api/v1/tests/tests_zones.py

```diff
@@ -543,7 +543,7 @@ def test_zones_ns_patch_400_bad_request(self):
                                   {'garbage': self.ns_one.name})
 
     @skip("Not testable, yet")
-    def test_zones_ns_patch_404_not_found(self):
+    def test_zones_ns_patch_404_not_found(self):  # pragma: no cover
```

Comment:
Why? Add comment or fix testing.

Reply:
The skipped issue again. I can make some wider comments on that.

Reply:
Fixed in https://github.com/unioslo/mreg/pull/484/commits/3fb4213c07bbd979d25a4cf18a83dae61abdb22b